MotionMark/tests/core/resources/multiply.js

```diff
@@ -24,120 +24,183 @@
  */
 (function() {
 
+class Tile {
```

Comment:
The rest of the code doesn't yet use `class` syntax so I'm not sure this should yet.

Reply:
The ES6 class is removed and `Utilities.createClass()` is used instead. However this way is not great. `Utilities.createClass()` allows only one level of inheritance. I wanted to add a second level of inheritance between Stage and MultiplyStage

Stage
    ReusableParticlesStage
        MultiplyStage

I wanted to emphasize that the elements of Multiply are reusable. They are shown and hidden instead of created and removed. But I could not do that because `Utilities.createClass()` extends the prototype of the subclass with the prototype of the superclass only. It does not do that recursively. There might be tricky way to still do it. But I could not figure it out.